packages/components/src/ui/styles/theme/config.js

```diff
@@ -0,0 +1,370 @@
+/**
+ * Internal dependencies
+ */
+import { get } from '../../create-styles';
```

Comment:
@jasmussen, who could help reviewing all those config values proposed here?

Reply:
To be quite honest, I don't recognize that many of them. The colors I do recognize seem to be quite out of date with the colors defined in our mixin, which are these:

<img width="885" alt="Screenshot 2021-04-09 at 09 57 21" src="https://user-images.githubusercontent.com/1204802/114148516-255d4100-991a-11eb-8cf2-f16706ea9143.png">

Not only that, but there are numerous additional colors which appear to not be used anywhere, have been created in isolation, don't sync up with the G2 designs from #18667, or even the the colors proposed for the current [default color schemes](https://make.wordpress.org/design/2020/09/18/project-reducing-colors-in-core/). If we do start using them, it will simply fragment and dilute the interface. I even see the "Inter" font introduced. It's not used anywhere. To frame it differently, anything we introduce should be based on the designs in #27331, and I'd appreciate us not introducing any fonts, colors, shadows or treatments not outlined there.